pkg/crc/config/callbacks_darwin.go

```diff
@@ -0,0 +1,24 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/code-ready/crc/pkg/os/launchd"
+	"github.com/spf13/cast"
+)
+
+var agentLabels = []string{"crc.tray", "crc.daemon"}
+
+func DisableEnableTrayAutostart(key string, value interface{}) string {
+	// Enable
+	if cast.ToBool(value) {
+		return RequiresCRCSetup(key, value)
+	}
+	// Disable
+	for _, agentLabel := range agentLabels {
+		if err := launchd.RemovePlist(agentLabel); err != nil {
+			return fmt.Sprintf("Error trying to disable auto-start of tray: %s", err.Error())
+		}
+	}
```

Comment:
Wondering if we should also leave this removal up to `crc setup` even if that's a bit odd?
Even if we decide to trigger the file removal from here, the code itself should be moved to a different package, `config` has no business knowing the tray uses 2 labels `crc.tray` `crc.daemon` which are managed using `launchd`.

Reply:
Moved this to a separate tray package `pkg/crc/tray`